client/src/components/ItemList/index.js

```diff
@@ -4,25 +4,33 @@ import Title from '../Shared/Title';
 import GButton from '../Shared/GreenButton';
 import Footer from '../Shared/Footer';
 import Item from '../Item';
-import { List, StyledHeader, StyledBottom, StyledLink, GButtonContainer } from './itemlist.style';
+import {
+  List, StyledHeader, StyledBottom, StyledLink, GButtonContainer,
+} from './itemlist.style';
+
 class ItemList extends Component {
   state = {
     itemlist: [],
   };
 
   componentDidMount() {
     axios.get('/items')
-      .then(res => this.setState({itemlist: res.data.data}))
-      .catch(err => console.log(err))
+      .then(res => this.setState({ itemlist: res.data.data }))
+      .catch(err => console.log(err));
   }
 
+  uploadPhoto = () => {
```

Comment:
the name of the function is tricky :p 
it would be better if its called goToUploadPage or just addNewItem

Reply:
Good point.